website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,49 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroy Infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Delete Workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all infrastructure managed by a workspace.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. It's recommended that you destroy the infrastructure managed by a workspace prior to deleting the workspace itself, otherwise the infrastructure resources will continue to exist but will become unmanaged.
+You will need to go into your infrastructure providers to delete the resources manually.
+
+Destroy plans
+
+Before queuing a destroy plan, you must enable the `Allow destroy plans` setting.
+
+### Manually Destroy
+
+Adds a destroy plan to the run queue.
+
+### Automatically Destroy
+
+<!-- BEGIN: TFC:only name:pnp-callout -->
+@include 'tfc-package-callouts/ephemeral-workspaces.mdx'
+<!-- END: TFC:only name:pnp-callout -->
+
+Schedules a destroy planx to be run at some point in the future.
+
+This value will be automatically unset after an auto destroy run has been queued.
```

Comment:
Can we provide guidance on what they do next? Do they need to reset the times of their other automatically queued runs? 
```suggestion
If you queue an automatic destroy run, the set times of your other currently queued scheduled runs is automatically unset.
```

Reply:
That makes sense to me!